utilities/copyoffload_migration.py

```
@@ -557,55 +703,112 @@ def _log_xcopy_verification_result(
+
+    Returns:
+        list[dict[str, str]]: List of pod logs with keys: pod_name, pvc_name, log_content.
+
+    Raises:
+        ValueError: If no populate pods found and no cached logs available.
+    """
+    # Try cached logs first (for MTV builds that cleanup pods quickly)
+    cached_logs: list[dict[str, Any]] | None = fixture_store.get("populate_pod_logs", {}).get(migration_uid)
+
+    if cached_logs:
+        LOGGER.info(f"Using {len(cached_logs)} cached populate pod log(s)")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)